### PR TITLE
Fix some more typos and cleanup

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

File: desktop/src/main/java/bisq/desktop/components/paymentmethods/data/JapanBankData.java [hunk 1]
```diff
@@ -355,7 +355,7 @@ private static String prettyPrintBank(String bankId, String bankName) // {{{
         .put("0562", "みなと銀行")
         .put("0565", "島根銀行")
         .put("0566", "トマト銀行")
-        .put("0569", "もみじ銀行")
+        .put("0569", "もみじ銀行")
```
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/components/paymentmethods/data/JapanBankData.java]: What did you change here? @wiz Is this correct?

Comment (rex4539) [reply desktop/src/main/java/bisq/desktop/components/paymentmethods/data/JapanBankData.java]: Hm, that was not intended. Will revert.

Comment (rex4539) [reply desktop/src/main/java/bisq/desktop/components/paymentmethods/data/JapanBankData.java]: Some weird encoding issue with editor. Fixed.

**Review by ripcurlx — APPROVED** [review]
utACK

**Comment by rex4539** [comment]
Also filed 3 more typo fix PRs in other repos, in case you missed them :)

https://github.com/bisq-network/incubator-bisq-xmr-integration/pull/2
https://github.com/bisq-network/incubator-bisq-api/pull/21
https://github.com/bisq-network/bisq-website/pull/289

**Review by RiccardoMasutti — APPROVED** [review]

**Pull request merged**